core/app/cview.class.php

```diff
@@ -27,7 +27,7 @@ public function __construct()
     private function init()
     {
         // Set to true to force template generation if a template has changed
-        $this->compile_check = false;
+        $this->compile_check = $config['compile_check'];
```

Comment:
No problem. I thought to use this variable because when is necessary to change some views, if this variable is not set to yes I can't see the changes